src/js/components/Menu/Menu.js

```diff
@@ -204,6 +204,7 @@ const Menu = props => {
         a11yTitle={messages.closeMenu || 'Close Menu'}
         active={activeItemIndex === controlButtonIndex}
         focusIndicator={false}
+        hoverIndicator="background"
```

Comment:
what happens if the user don't want this hover behavior? shouldn't we allow it to be customized instead of forcing it at this point?

Reply:
oh wait, gotcha, it is only for the Button which is currently missing the hoverIndicator behavior.

Reply:
Yes, just the control button when it is included in the drop with the menu items, so the hoverIndicator here will match the hover indicator that appears on the other menu items as well. I think it's important to consider though, would there be a case where someone wouldn't want a hover on this button? I think from a consistency in behavior across the drop, @ericsoderberghp and I felt that it should be receiving hover indication but I'm open to discussing it further.